flams/run_blast.py

```diff
@@ -69,6 +74,15 @@ def _run_blast(input, modification, lysine_pos, lysine_range, evalue, num_thread
     return [_filter_blast(i, lysine_pos, lysine_range, evalue) for i in blast_records]
 
 
+# BlastpCommandline does not like whitespaces in paths, which the database may contain on especially Mac OS.
+# Therefore, we will change our working directory to that of the data dir before running Blast.
+# Before doing this, we need to convert the relative path to the input file into an absolute path.
+def _adjust_working_directory(input: str):
+    input = os.path.abspath(input)
```

Comment:
I'd rather stick to pathlib.Path: https://docs.python.org/3/library/pathlib.html. It deals with multiple OS, allows for easy moving around directories and passing a Path object is much clearer than just a string.

Reply:
it was actually already using Path, so only the typing here was wrong